GOVERNANCE_DOC.md

```diff
@@ -0,0 +1,619 @@
+  
+## Proposal submission
+
+### Right to submit a proposal
+
+Any Atom holder, whether bonded or unbonded, can submit proposals by sending a `TxProposal` transaction. Once a proposal is submitted, it is identified by its unique `proposalID`.
+
+### Proposal filter (minimum deposit)
+
+To prevent spam, proposals must be submitted with a deposit in Atoms. Voting period will not start as long as the proposal's deposit is inferior to the minimum deposit parameter `MinDeposit`.
+
+When a proposal is submitted, it has to be acompagnied by a deposit that must be strictly positive but that can be inferior to `MinDeposit`. Indeed, the submitter need not pay for the entire deposit on its own. If a proposal's deposit is strictly inferior to `MinDeposit`, other Atom holders can increase the proposal's deposit by sending a `TxDeposit` transaction. Once the proposals's deposit reaches `minDeposit`, it enters voting period. 
+
+### Deposit refund
+
+There are two instances where Atom holders that deposited can claim back their deposit:
+- If the proposal is accepted
+- If the proposal's deposit does not reach `MinDeposit` for a period longer than `mMxDepositPeriod` (initial value: 2 months). Then the proposal is considered closed and nobody can deposit on it anymore.
+
+In such instances, Atom holders that deposited can send a `TxClaimDeposit` transaction to retrieve their share of the deposit.
+
+### Linked proposals
+
+When an Atom holder submits a proposal, it can link it to another proposal by specifying an existing `proposalID`. It can be useful in cases where the proposal is a fix or upgrade of a past proposal (for example).
+
+### Proposal types
+
+In the initial version of the governance module, there are two types of proposal:
+- `PlainTextProposal`. All the proposals that do not involve a modification of the source code go under this type. For example, an opinion poll would use a proposal of type `PlainTextProposal`
+- `SoftwareUpgradeProposal`. If accepted, validators are expected to update their software in accordance with the proposal. They must do so by following a 2-steps process described in the **Software Upgrade** section below. Software upgrade roadmap may be discussed and agreed on via `PlainTextProposals`, but actual software upgrades must be performed via `SoftwareUpgradeProposals`.
+
+### Proposal categories
+
+There are two categories of proposal:
+- `Regular`
+- `Urgent`
+
+These two categories are strictly identical except that `Urgent` proposals can be accepted faster if a certain condition is met. For more information, see **Thresholds** section.
+
+
+## Vote
+
+### Participants
+
+*Participants* are users that have the right to vote. On the Cosmos Hub, participants are bonded Atom holders. Unbonded Atom holders and other users do not get the right to participate in governance. However, they can submit and deposit on proposals.
```

Comment:
Can you deposit bonded atoms on a proposal? The answer should probably be no.

Reply:
No